cloud_governance/policy/common_policies/send_aggregated_alerts.py

```diff
@@ -22,143 +14,188 @@ class SendAggregatedAlerts:
     This class send alerts to users which conditions are not satisfied by the policies
     """
 
-    FILE_NAME = 'resources.json'
-    GLOBAL_REGION = 'us-east-1'
-    TODAY_DATE = str(date.today()).replace('-', '/')
-
     def __init__(self):
         self.__environment_variables = environment_variables.environment_variables_dict
-        self.__bucket_name = self.__environment_variables.get('BUCKET_NAME')
-        self.__bucket_key = self.__environment_variables.get('BUCKET_KEY')
-        self.__policies = self.__environment_variables.get('POLICIES_TO_ALERT')
-        self.__s3_operations = S3Operations(region_name='us-east-2', bucket=self.__bucket_name, logs_bucket_key=self.__bucket_key)
-        self.__active_regions = EC2Operations().get_active_regions()
-        self.__kerberos_users = self.__get_kerberos_users_for_iam_users()
-        self.__global_region_policies = ['s3-inactive', 'empty-roles']
-        self.__mail_alert_days = self.__environment_variables.get('MAIL_ALERT_DAYS')
-        self.__policy_action_days = self.__environment_variables.get('POLICY_ACTIONS_DAYS')
+        self.__days_to_delete_resource = int(self.__environment_variables.get('DAYS_TO_DELETE_RESOURCE'))
+        self.__mail_to = self.__environment_variables.get('EMAIL_TO')  # testing purposes
+        self.__mail_cc = self.__environment_variables.get('EMAIL_CC', [])
         self.__mail_message = MailMessage()
```

Comment:
Why we need EMAIL_TO and EMAIL_CC, do we use it anywhere ?

Reply:
It forcefully replaces to and cc. I used this variable to work on local env and running in dev containers.